multitenancy/multitenancy-support-in-gitops.adoc

```diff
@@ -0,0 +1,62 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="multitenancy-support-in-gitops"]
+= Multitenancy support in GitOps
+:context: multitenancy-support-in-gitops
+
+toc::[]
+
+Multitenancy is a software architecture where a single software instance serves multiple distinct user groups, or tenants. Using multitenancy you can share a single Argo CD instance to deploy resources while maintaining isolation your between users. This section is useful to cluster administrators, and provides an understanding of Argo CD instance scopes and when to choose a particular mode.
+
+With {gitops-title} Operator on {OCP}, as a cluster administrator, you can provide multitenant access to the cluster for application delivery teams (tenants). You can allow tenants to create and manage dedicated Argo CD instances in their user-defined namespaces without having administrative permissions. The tenants have full autonomy and can tailor this instance to their needs and requirements without interfering with other tenants.
+
+[NOTE]
+====
+For a multitenant cluster, the person managing the Argo CD varies from the person who manages the cluster and its users. Therefore, you cannot have the Argo CD Application Controller (`argocd-application-controller` component) as a super-user in the cluster.
+====
+
+Because Argo CD Application Controller is responsible for the reconciliation of resources in the managed clusters, to support multitenancy in {gitops-shortname} Operator, depending on your use cases, tenants, and requirements, you must configure, grant, extend, or restrict specific permissions explicitly to Argo CD instances, applications, and on remote clusters to perform certain operations.
```

Comment:
This sentence is quite long. Consider breaking it up. Maybe something like this:

```suggestion
The Argo CD Application Controller reconciles resources in the managed clusters. Therefore, to use multitenancy in the {gitops-shortname} Operator, you must set permissions explicitly to Argo CD instances, applications, and on remote clusters to perform certain operations.
```

Reply:
Thanks for the feedback. I have implemented the 1st half of it.
I prefer to retain the last part as is.